change/@fluentui-react-menu-30e18336-5048-4a09-ac73-8df822fa4336.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Updates getTriggerChild internal usage",
```

```suggestion
  "comment": "fix: improve internal typings",
```

_nit_ I think that it's more relevant to the change